unit-tests/ModelsRelationsTest.php

```diff
@@ -374,6 +374,13 @@ protected function _testIssue938($di)
 		}
 	}
 
+	protected function _testIssue10800()
```

Comment:
This method here is not needed. In future `unit-tests` folder will be removed in favor of `tests`